### PR TITLE
[WIP] Subscriptions redirect (updated for master)

### PR DESCRIPTION
**Pull request by ianb**
This is an attempt to recreate #1310 against master. From the previous PR:

- add `/subscriptions/*` path to server and client-side router

- SubscriptionsRedirectView that issues an access token and redirects to
  a path on fxa-payments-server

Issue #926

**Note:** Right now this does not appear to actually work locally, maybe it reintroduces #1637, or maybe it's another issue.

### CODE REVIEW

File: packages/fxa-content-server/app/scripts/views/subscriptions_product_redirect.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import FormView from './form';
+import Template from 'templates/subscriptions_redirect.mustache';
+import PaymentServer from '../lib/payment-server';
+
+class SubscriptionsProductRedirectView extends FormView {
+  mustAuth = true;
+  template = Template;
+
+  initialize (options) {
```
Comment (bbangert) [inline packages/fxa-content-server/app/scripts/views/subscriptions_product_redirect.js]: I thought prettier would remove the space here, or does that not run on content-server yet?

File: packages/fxa-payments-server/server/lib/server.js [hunk 2]
```diff
@@ -2,25 +2,25 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-'use strict';
+"use strict";
```
Comment (bbangert) [inline packages/fxa-payments-server/server/lib/server.js]: I thought our prettier spec would have these all stay single quote?

Comment (ianb) [reply packages/fxa-payments-server/server/lib/server.js]: I'm a little confused what happened with the style changes too. I worry all the rebasing and conflict merging might have left things a little screwy. I should probably just re-prettify.

Comment (ianb) [reply packages/fxa-payments-server/server/lib/server.js]: I'm not clear what is being applied, but running prettier makes an even bigger diff than this, so best to hold on that.

**Comment by ianb** [comment]
I've decided to manually recreate #1310 on top of master.

**Pull request closed without merging**